.github/workflows/bb-forge-test.yml

```
@@ -0,0 +1,161 @@
+          cache-path: ${{ inputs.cache-path }}
+          fail-on-cache-miss: true
+
+      - name: "Run the tests"
+        working-directory: ${{ inputs.working-directory }}
+        run: 'forge test --match-path "${{ inputs.match-path }}"'
+        continue-on-error: true
+
+      - name: "Run BB Action CI"
+        uses: BuildBearLabs/buildbear_x_action@v1.7.3
+        with:
+          buildbear-api-key: ${{ secrets.BUILDBEAR_API_KEY }}
```

actionlint (1.7.10)

[error] property "buildbear_api_key" is not defined in object type {actions_runner_debug: string; actions_step_debug: string; api_key_infura: string; github_token: string; mainnet_rpc_url: string}

(expression)